examples/calibration/plot_cutoff_calibration.py

```diff
@@ -0,0 +1,154 @@
+"""
+=======================================================================
+Decision threshold (cutoff point) calibration for binary classification
+=======================================================================
+
+Most machine learning classifiers that offer probability estimation do so by
```

Comment:
I'm not sure it's fair to frame this in terms of probability. Better to say that binary predictions are often based on real-valued decision functions, with the threshold set to optimise...